### DISCUSSION TITLE
Error While trying to update repo

### DISCUSSION THREAD

**Discussion by gameboin** [TheLastBen/fast-stable-diffusion#412] [Q&A]
I've Been getting this error for a few days, was kind of hoping it would fix itself after another update, but apparently not. 
Not sure what it means by commit your changes or stash them before you merge?

My error msg when trying to update repo:

`error: Your local changes to the following files would be overwritten by merge:
	modules/processing.py
Please commit your changes or stash them before you merge.
Aborting`

**Comment by TheLastBen** [comment]
run this in a new cell : 
`!rm /content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/processing.py`

then update the repo
